egs/babel_universal_am/s5/local/convert_dict.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2016  Johns Hopkins University (Author: Matthew Wiesner)
+# Apache 2.0
+
+# This script presupposes a manual mapping from the missing phones to existsing
+# phones already exists.
+
+
+if [ $# -lt 6 ]; then
+  echo >&2 "Usage: ./local_/convert_lang.sh <odict> <olang> <idict1> <idict2> <ilang> <map>"
+  exit 1
```

Comment:
I think you should be explaining at least conceptually what the individual script parameters mean